assets/scss/7-layout/content-grid.html

```diff
@@ -0,0 +1,28 @@
+<div class="l-content-grid">
+  <div class="l-content-grid__content has-bg-white-off">
+    <h3 class="has-padding">Content</h3>
+  </div>
+  <div class="l-content-grid__side has-bg-gray-light">
+    <h3 class="has-padding">Sidebar</h3>
+  </div>
+</div>
+
+<hr>
+
+<p>Example with overrides</p>
+<style>
+/* Scope to a variation of your choosing */
+.l-content-grid--custom {
+  --content-grid-order: 'side content';
+  --content-grid-template: 20rem 1fr;
+  --content-grid-gap: 0;
+}
```

Comment:
@AndrewGibson27 for example in your UMP grid, if you used this helper, you could just add those five lines to https://github.com/texastribune/donations/blob/master/static/sass/ds/8-layout/_grid.scss